guides/source/action_view_overview.md

```diff
@@ -42,34 +44,37 @@ $ bin/rails generate scaffold article
-The complete HTML returned to the client is composed of a combination of this ERB file, a layout template that wraps it, and all the partials that the view may reference. Within this guide, you will find more detailed documentation about each of these three components.
+The naming convention is that the views share their name with the associated controller action. For example the `index.html.erb`, `edit.html.erb`, etc. above.
+
+By following this naming convention, Rails will automatically find and render the matching view at the end of a controller action, without you having to  specify it. For example, the `index` action in the `articles_controller.rb` will automatically render the `index.html.erb` view inside the `app/views/articles/` directory. The name and the location of the file are both important.
 
-As mentioned, the final HTML output is a composition of three Rails elements: `Templates`, `Partials` and `Layouts`.
-Below is a brief overview of each of them.
+The final HTML returned to the client is composed of a combination of the `.html.erb` ERB file, a layout template that wraps it, and all the partials that the ERB file may reference. In the rest of this guide, you will find more details about each of the three components: `Templates`, `Partials`, `Layouts`.
 
 Templates
 ---------
 
-Action View templates can be written in several ways. If the template file has a `.erb` extension then it uses a mixture of ERB (Embedded Ruby) and HTML. If the template file has a `.builder` extension then the `Builder::XmlMarkup` library is used.
+Action View templates can be written in different formats. If the template file has a `.erb` extension, it uses embedded Ruby to build a HTML response. If the template has a `.jbuilder` extension, it uses the [Jbuilder](https://github.com/rails/jbuilder) gem to build a JSON response. And a template with a `.builder` extension uses the [`Builder::XmlMarkup`](https://github.com/jimweirich/builder) library to build an XML response.
 
-Rails supports multiple template systems and uses a file extension to distinguish amongst them. For example, an HTML file using the ERB template system will have `.html.erb` as a file extension.
+Rails uses the file extension to distinguish among multiple template systems. For example, an HTML file using the ERB template system will have `.html.erb` as a file extension.
 
 ### ERB
 
-Within an ERB template, Ruby code can be included using both `<% %>` and `<%= %>` tags. The `<% %>` tags are used to execute Ruby code that does not return anything, such as conditions, loops, or blocks, and the `<%= %>` tags are used when you want output.
+When Rails processes the ERB view templates ending with `.html.erb`, it evaluates the embedded Ruby code and replaces the ERB tags with their respective outputs. The dynamic content generated by evaluating the embedded Ruby code is combined with the static HTML markup to form the final HTML response.
```

Comment:
This section can be a lot more simple-speak. I remember in our bootcamp my teacher explaining it something like (paraphrasing): "An ERB template is a way to sprinkle a bit of Ruby code in amongst static HTML by using `<% %>` and `<%= %>` tags. 
The `<% %>` tags are used for embedding Ruby code without directly outputting the result, while `<%= %>` tags are used when you want to show the output of the code on the page."